core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -468,6 +468,10 @@ private JdbcUtil() {}
 
   static Namespace stringToNamespace(String namespace) {
     Preconditions.checkArgument(namespace != null, "Invalid namespace %s", namespace);
+    if (namespace.isEmpty()) {
```

Comment:
previously this method would return the identifier as `.tblName` instead of `tblName` when using an empty namespace, because L475 passes an array with a single empty string to `Namespace.of()`. 

I guess the question here is: is `Namespace.of("")` the same as `Namespace.empty()` and should we maybe fix `Namespace.of()`?
It seems there are currently no existing tests that would indicate what the correct behavior should be in this case

Reply:
I think they should be treated the same, I also hope it doesn't come up very often